R/tribble.R

```diff
@@ -19,6 +30,26 @@
 #'   "c",   3
 #' )
 #'
+#' # Conversion expressions can be supplied, including additional
+#' # paremeters if needed.
+#' tribble(
+#'   ~colA, colB~factor(., levels = c("B", "A")),
```

Comment:
Would it be difficult to make the dot optional, à la magrittr? We might even be able to offload the entire heavy lifting to magrittr (which we are importing anyway) with some computing on the language.